redbot/cogs/downloader/downloader.py

```diff
@@ -1388,20 +1388,20 @@ async def findcog(self, ctx: commands.Context, command_name: str) -> None:
             cog_name = self.cog_name_from_instance(cog)
             installed, cog_installable = await self.is_installed(cog_name)
             if installed:
-                is_installable = True
                 made_by = humanize_list(cog_installable.author) or _("Missing from info.json")
                 repo_url = (
                     _("Missing from installed repos")
                     if cog_installable.repo is None
                     else cog_installable.repo.clean_url
                 )
                 cog_name = cog_installable.name
-            else:
-                # Assume it's in a base cog
-                is_installable = False
-                made_by = "26 & co."
+            elif cog.__module__.startswith("redbot."):  # core commands or core cog
+                made_by = "Cog Creators"
                 repo_url = "https://github.com/Cog-Creators/Red-DiscordBot"
                 cog_name = cog.__class__.__name__
+            else:  # assume not installed via downloader
+                made_by = _("Unknown")
+                repo_url = _("None - this cog wasn't installed via downloader")
```

Comment:
This should use the same thing for `cog_name` as it did before the change.
```suggestion
                repo_url = _("None - this cog wasn't installed via downloader")
                cog_name = cog.__class__.__name__
```